### ISSUE TITLE
Why can Project Admins not get permission on project dashboards?

### ISSUE THREAD

**Issue by KarstenSchaeffer** [MicrosoftDocs/azure-devops-docs#10161]
Related to section "Set permissions for a project dashboard":
The owner of a project dashboard can give permissions to users or groups, but NOT for the project's Project Administrators. When selecting this group, the permission drop-downs are read-only with value "Not set".
Why does a Project Admin not have by default permissions on the project dashboard?
If this is intended (not a bug), please add a hint in the doc.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: b3a8994c-97ff-1347-d621-66ae5e9c11ce
* Version Independent ID: 7e558469-28b6-bade-4ff0-4e2bf9a24e85
* Content: [Set dashboard permissions for team members - Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/report/dashboards/dashboard-permissions?view=azure-devops)
* Content Source: [docs/report/dashboards/dashboard-permissions.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/report/dashboards/dashboard-permissions.md)
* Product: **devops**
* Technology: **devops-analytics**
* GitHub Login: @KathrynEE
* Microsoft Alias: **kaelli**

**Comment by KathrynEE** [comment]
@KarstenSchaeffer - Thank you for your inquiry. I'm checking with the product team. It appears to be a product bug. I'll wait to hear back from them before taking action on updating the article. 

As I looked into this, I found that I could create a custom security group in Azure DevOps at the org level and then add different Project Administrator groups for different projects. I found that I could add this to a dashboard security dialog and change the permissions. So this is a workaround for the time being. 

Again, many thanks for letting us know of this problem.
